Maddori.Apple/Maddori.Apple/Screen/MyFeedback/MyFeedback/Cell/MyFeedbackMemberCollectionViewCell.swift

```diff
@@ -48,35 +54,51 @@ final class MyFeedbackMemberCollectionViewCell: BaseCollectionViewCell {
     override func render() {
         self.addSubview(shadowView)
         shadowView.snp.makeConstraints {
+            $0.centerX.top.equalToSuperview()
+            $0.width.height.equalTo(60)
+        }
+        
+        shadowView.addSubview(profileImageView)
+        profileImageView.snp.makeConstraints {
             $0.edges.equalToSuperview()
         }
         
         self.addSubview(memberNameLabel)
         memberNameLabel.snp.makeConstraints {
-            $0.center.equalToSuperview()
+            $0.top.equalTo(shadowView.snp.bottom).offset(8)
             $0.leading.trailing.equalToSuperview().inset(4)
         }
     }
     
     // MARK: - func
     
     private func setSelectedCellLayer() {
-        self.layer.cornerRadius = 30
-        self.layer.borderColor = UIColor.blue200.cgColor
-        self.layer.borderWidth = 2
-        self.layer.shadowOpacity = 0
+        shadowView.layer.cornerRadius = 30
+        shadowView.layer.borderColor = UIColor.blue200.cgColor
+        shadowView.layer.borderWidth = 2
+        shadowView.layer.shadowOpacity = 0
         memberNameLabel.textColor = .blue200
     }
     
     private func setShadowLayer() {
-        self.layer.cornerRadius = 30
-        self.layer.borderWidth = 0
-        self.layer.borderColor = UIColor.clear.cgColor
-        self.makeShadow(color: .black, opacity: 0.25, offset: CGSize.zero, radius: 1)
+        shadowView.layer.cornerRadius = 30
+        shadowView.layer.borderWidth = 0
+        shadowView.layer.borderColor = UIColor.clear.cgColor
+        shadowView.makeShadow(color: .black, opacity: 0.25, offset: CGSize.zero, radius: 1)
         memberNameLabel.textColor = .gray400
     }
     
     func setMemberName(name: String) {
         memberNameLabel.text = name
     }
+    
+    func setMemberProfileImage(from path: String?) {
+        if let path = path {
```

Comment:
p4;

```suggestion
        if let path {
```

위와 같이 변경할 수 있습니당~~

Reply:
[9dab9ff](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/317/commits/9dab9ff8e7ee0bf84310450dba12c0de17dc428e)

왁 최고 ^_______^